### PR TITLE
JET-333 Heap-size flags

### PR DESCRIPTION
**Pull request by fsalazarYW**
Explicitly give starting and max size of heap. Hopefully will prevent heroku restarts when we grow past our size allotment.

@gurpreet-shah @cashman ... thanks.

### CODE REVIEW

File: scripts/heroku-run.bash [hunk 1]
```diff
@@ -46,6 +46,7 @@ function putS3
 }
 
 java -ea \
+  -Xms2400m -Xmx2400m
```
Comment (gurpreet-shah) [inline scripts/heroku-run.bash:49]: so we are doing `min == max`?

Comment (fsalazarYW) [reply scripts/heroku-run.bash:49]: Yes, no need to grow up to max, we know we will need at least this much heap.

**Review by gurpreet-shah — COMMENTED** [review]
lgtm

File: scripts/heroku-run.bash (hunk 1 above)
Comment (cashman) [reply scripts/heroku-run.bash:49]: Does this value include what Heroku reports as "cache" as well as "rss"?  The sum of those is what their quota is based on.  From the logs, looks like our "cache" value tops out around 160MB.

Consider setting this a little lower, if we want to be sure?

**Pull request merged**